### ISSUE TITLE
add state toggle for spells and skills

### ISSUE THREAD

**Issue by outofnamespace** [richardwilkes/gcs#622]
This would be handy for point change effects for PC/NPC and future growth plans for a PC. It is quite useful for traits. I have all racial traits in containers. Toggle the container off to see the point impact of being an elf, etc.

**Comment by richardwilkes** [comment]
For race, I'd recommend using the Race trait container type. This assigns all points from that container and its children to the "Race" category in the point breakdown.

**Comment by TheFabulousIronChef** [comment]
Yeah, that works well for advantages and is a good suggestion. For skills
and spells, right now, I just add them, reduce the point total so they are
zero points, and put them in a container that is called "advancement" or
"spells to get" or something like that. It's not as elegant as a toggle
state, but it works until that's available.

**Comment by richardwilkes** [comment]
I don't see a need for this that isn't covered by other options already available. If I'm missing the point, feel free to reopen and provide more specific details.
